packages/express-backend/services.js

```diff
@@ -83,7 +83,7 @@ function getChores() {
 function addChore(chore) {
   const choreToAdd = new choreModel(chore);
   const promise = choreToAdd.save();
-  return promise;
+  return promise.catch((error) => console.log("Error", error));
```

Comment:
Does this still return promise if there are no errors? Just checking.

Reply:
I believe so - the professor added it, saying that it should allow us to see what exact error it's throwing. Sadly, since it seems to be a server error with Azure, we can't see exactly what's going on.

Reply:
Did he suggest any way that I can look into this on the azure side?